services/nexus/nexus.service.js

```diff
@@ -46,14 +46,13 @@ module.exports = class Nexus extends BaseJsonService {
       base: 'nexus',
       // API pattern:
       // /nexus/(r|s|<repo-name>)/(http|https)/<nexus.host>[:port][/<entry-path>]/<group>/<artifact>[:k1=v1[:k2=v2[...]]]
-      format:
-        '(r|s|[^/]+)/(https?)/((?:[^/]+)(?:/[^/]+)?)/([^/]+)/([^/:]+)(:.+)?',
-      capture: ['repo', 'scheme', 'host', 'groupId', 'artifactId', 'queryOpt'],
+      pattern:
+        ':repo(r|s|[^/]+)/:scheme(http|https)/:hostAndPath+/:groupId/:artifactId([^/:]+):queryOpt(:.+)?',
```

Comment:
I'm updating the `hostAndPath` param name in the examples too as I think it'll be helpful for folks to see that in the editor window